e2e_tests/tests/cluster/utils.py

```diff
@@ -20,10 +20,10 @@ def cluster_slots() -> Dict[str, Any]:
     # TODO: refactor tests to not use cli singleton auth.
     certs.cli_cert = certs.default_load(conf.make_master_url())
     authentication.cli_auth = authentication.Authentication(conf.make_master_url(), try_reauth=True)
-    r = api.get(conf.make_master_url(), "agents")
+    r = api.get(conf.make_master_url(), "api/v1/agents")
```

Comment:
usually i'd say we should opportunistically swap to the python bindings for the new API, but I dont wanna hold you up. just for next time.